ccl/changefeedccl: TestChangefeedSchemaChangeBackfillCheckpoint failed

ccl/changefeedccl.TestChangefeedSchemaChangeBackfillCheckpoint [failed](https://tanzanite.cluster.engflow.com/invocations/default/5bc5a6cd-5eca-431c-8078-991fc6a32a35?testReportRun=10&testReportShard=11&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on release-24.2 @ [cf9b1574fd03c877cc831f4be41a5e12b1055a42](https://github.com/cockroachdb/cockroach/commits/cf9b1574fd03c877cc831f4be41a5e12b1055a42):


```
=== RUN   TestChangefeedSchemaChangeBackfillCheckpoint
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestChangefeedSchemaChangeBackfillCheckpoint4096890465
    test_log_scope.go:76: use -show-logs to present logs inline
    changefeed_test.go:2250: random seed: 8069558234357739010
    changefeed_test.go:2521: -- test log scope end --
test logs left over in: outputs.zip/logTestChangefeedSchemaChangeBackfillCheckpoint4096890465
--- FAIL: TestChangefeedSchemaChangeBackfillCheckpoint (51.43s)
=== RUN   TestChangefeedSchemaChangeBackfillCheckpoint/pubsub
    helpers_test.go:872: making server as system tenant
    helpers_test.go:948: making pubsub feed factory
    helpers_test.go:65: using legacy schema changer
    changefeed_test.go:2316: highwater: 1728804604.847654689,0
    changefeed_test.go:2361: backfill timestamp: 1728804606.936638921,2147483647
    changefeed_test.go:2408: initial checkpoint: [/Table/106/1/{101/0-501/0}]
    changefeed_test.go:2467: condition failed to evaluate within 45s: from changefeed_test.go:2471: waiting for second checkpoint
    --- FAIL: TestChangefeedSchemaChangeBackfillCheckpoint/pubsub (51.41s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=10</code>
 - <code>shard=11</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedSchemaChangeBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-43136

ccl/changefeedccl.TestChangefeedSchemaChangeBackfillCheckpoint [failed](https://tanzanite.cluster.engflow.com/invocations/default/c97bd269-308c-4323-b5af-8599b6649fcc?testReportRun=18&testReportShard=11&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on release-24.2 @ [52fafd7ae8a078124f983d0c60550e9f38c68462](https://github.com/cockroachdb/cockroach/commits/52fafd7ae8a078124f983d0c60550e9f38c68462):


```
=== RUN   TestChangefeedSchemaChangeBackfillCheckpoint
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestChangefeedSchemaChangeBackfillCheckpoint3671982161
    test_log_scope.go:76: use -show-logs to present logs inline
    changefeed_test.go:2250: random seed: -8238013932062361849
    changefeed_test.go:2521: -- test log scope end --
test logs left over in: outputs.zip/logTestChangefeedSchemaChangeBackfillCheckpoint3671982161
--- FAIL: TestChangefeedSchemaChangeBackfillCheckpoint (52.42s)
=== RUN   TestChangefeedSchemaChangeBackfillCheckpoint/cloudstorage
    helpers_test.go:872: making server as system tenant
    helpers_test.go:948: making cloudstorage feed factory
    helpers_test.go:65: using legacy schema changer
    testfeed_test.go:301: creating external connection
    testfeed_test.go:304: ran create external connection
    changefeed_test.go:2316: highwater: 1728981421.362862902,0
    changefeed_test.go:2361: backfill timestamp: 1728981423.442265721,0
    changefeed_test.go:2408: initial checkpoint: [/Table/106/1/{101/0-501/0}]
    changefeed_test.go:2467: condition failed to evaluate within 45s: from changefeed_test.go:2471: waiting for second checkpoint
    --- FAIL: TestChangefeedSchemaChangeBackfillCheckpoint/cloudstorage (52.41s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=18</code>
 - <code>shard=11</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedSchemaChangeBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

--- a/pkg/ccl/changefeedccl/changefeed_test.go
+++ b/pkg/ccl/changefeedccl/changefeed_test.go
@@ -2352,7 +2352,7 @@ func TestChangefeedSchemaChangeBackfillCheckpoint(t *testing.T) {
 
 		// Checkpoint progress frequently, and set the checkpoint size limit.
 		changefeedbase.FrontierCheckpointFrequency.Override(
-			context.Background(), &s.Server.ClusterSettings().SV, 10*time.Millisecond)
+			context.Background(), &s.Server.ClusterSettings().SV, 1)
 		changefeedbase.FrontierCheckpointMaxBytes.Override(
 			context.Background(), &s.Server.ClusterSettings().SV, maxCheckpointSize)
 

